### ISSUE TITLE
Error in display of meta in ServiceStack/Templates/auth.html

### ISSUE THREAD

**Issue by titobrasolin-ke** [ServiceStack/Issues#732]
The line below seems a little strange to me:
https://github.com/ServiceStack/ServiceStack/blob/46b4578bf1a3c4b863ac9e1a960b92f329bb581e/src/ServiceStack/Templates/auth.html#L254

```javascript
if (cmpKey === 'meta')
{
    for (var name in v) {
        if (name.endsWith('-tokens')) {
            // ... Do something...
        } else {
            sb.push('<tr><th><div>' + humanize(k) + '</div></th><td><div>' + str + '</div></td></tr>');
        }
    }
}
```

If `name` does _not_ end with '-tokens' then `str` is printed, whatever it contains.
In fact, the "auth" page of the project I'm working on repeatedly shows the list of permissions next to the word "meta".

**Comment by mythz** [comment]
Can you please show a screenshot of a page containing the issue?

**Comment by titobrasolin-ke** [comment]
Of course, here it is:

![image](https://user-images.githubusercontent.com/64349766/88170113-6e028c00-cc1d-11ea-8895-6f1fa90a1a59.png)

**Comment by mythz** [comment]
I've updated it to render the meta properties in [this commit](https://github.com/ServiceStack/ServiceStack/commit/e72e5fc879c29632ee3410dc0a74c927c500abca).

This change is available from the latest v5.9.1 that's now [available on MyGet](https://docs.servicestack.net/myget). 

Can you post an updated screenshot after you've upgraded to the latest version (if it doesn't contain any sensitive info).

**Comment by titobrasolin-ke** [comment]
Thank you so much @mythz 

![image](https://user-images.githubusercontent.com/64349766/88266988-7536a200-ccd0-11ea-93c9-3f059fc7f4c7.png)
